tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -81,19 +81,32 @@ jobs:
 
 - template: templates/win-ci.yml
   parameters:
-    ort_build_pool_name: 'onnxruntime-gpu-winbuild'
+    ort_build_pool_name: 'onnxruntime-gpu-tensorrt8-winbuild'
     DoCompliance: ${{ parameters.DoCompliance }}
     DoEsrp: ${{ parameters.DoEsrp }}
     job_name_suffix: gpu
-    EnvSetupScript: setup_env_cuda_11.bat
+    EnvSetupScript: setup_env_gpu.bat
     buildArch: x64
     msbuildPlatform: x64
     packageName: gpu-x64
-    buildparameter: --use_cuda --cuda_version=11.1 --cuda_home="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.1" --cudnn_home="C:\local\cudnn-11.1-windows-x64-v8.0.5.39\cuda" --enable_onnx_tests --enable_wcos --build_java --cmake_extra_defines "CMAKE_CUDA_ARCHITECTURES=37;50;52;60;61;70;75;80"
+    buildparameter: --use_cuda --cuda_version=11.4 --cuda_home="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.4" --cudnn_home="C:\local\cudnn-11.4-windows-x64-v8.2.2.26\cuda" --enable_onnx_tests --enable_wcos --build_java --cmake_extra_defines "CMAKE_CUDA_ARCHITECTURES=37;52;60;61;70;75;80"
```

Comment:
I see, You removed 50 so it is smaller.

But, if 50 is not supported, 37 is still in the list?

Reply:
https://github.com/microsoft/onnxruntime/pull/8687#discussion_r687009568

"52 covers m60 , 50 is no longer supported i believe." as George mentioned.
Not quite sure about 37.

Reply:
I was wrong.  50 and 37 are deprecated but still supported. We should keep 37 because that's K80 and still used by various teams. We can continue to leave off 50 because that's M10 (nobody uses) we use M60 instead (52).
In the next major CUDA release, we can drop 37.